test/fixtures/cheatsheets.cheatmd

```diff
@@ -0,0 +1,763 @@
+---
+title: Elixir
+category: Elixir
+layout: 2017/sheet
+tags: [New]
+updated: 2018-07-04
+weight: -10
+---
```

Comment:
Do we suppose to do something with this?

Reply:
No, we won't have preludes I think. We can discard it :)